Marlin/Marlin_main.cpp

```diff
@@ -5696,6 +5828,9 @@ void home_all_axes() { gcode_G28(true); }
                 t_delta[A_AXIS] = (       - Z3(5) + Z3(9)         - Z3(11) + Z3(3)) * a_factor;
                 t_delta[B_AXIS] = ( Z3(1)         - Z3(9) + Z3(7)          - Z3(3)) * a_factor;
                 t_delta[C_AXIS] = (-Z3(1) + Z3(5)         - Z3(7) + Z3(11)        ) * a_factor;
+                e_delta[A_AXIS] += (t_delta[B_AXIS] - t_delta[C_AXIS])/4.5;
```

Comment:
add this to prevent to much wobble on the delta radius (level the saddle shape from angle change)